cmd/libs/go2idl/openapi-gen/README

```diff
@@ -0,0 +1,10 @@
+# Generate OpenAPI definitions
+
+To generate a definition for an specific type of package add "+openapi=$VALUE" tag to the type/package comment lines.
+$VALUE can be any of these:
+
+- "package": Generate spec for all types in this package (should only be added to package comments)
+- "definition": Generate spec for a single type (should only be added to type comments)
```

Comment:
optional: the convention used in deepcopy and conversion is that a "package" tag can be overriden at the type, and the values are true or false.  I would be fine to see "true, false, should-fail" ?